tests/snippets/builtin_complex.py

```diff
@@ -93,6 +93,8 @@
 assert hash(complex(3.14)) == hash(float(3.14))
 assert hash(complex(-float('inf'))) == hash(-float('inf'))
 assert hash(1j) != hash(1)
+assert hash(3.1 - 4.2j) == -1152921505020448009
```

Comment:
Is using a fixed value to test hashes a good idea? If I understand [the docs](https://docs.python.org/3/reference/datamodel.html#object.__hash__) correctly, the value might and will be different on different platforms:

> **Note:** `hash()` truncates the value returned from an object’s custom `__hash__()` method to the size of a `Py_ssize_t`. This is typically 8 bytes on 64-bit builds and 4 bytes on 32-bit builds. If an object’s `__hash__()` must interoperate on builds of different bit sizes, be sure to check the width on all supported builds. An easy way to do this is with `python -c "import sys; print(sys.hash_info.width)"`.

Reply:
@silmeth Yes, That makes sense Do you have any good ideas?

Reply:
If we just want to check whether the hash function of complex is crashed or not
````
python
hash(3.1 - 4.2j)
````
this statement will be enough

WDYT?

Reply:
I think I’d just leave the `hash()` statement alone like you suggested, or alternatively just check that it produces the same value twice (`assert hash(3.1 - 4.2j) = hash(3.1 - 4.2j)`).